samples/powerpoint/text/get-set-textrange.yaml

```diff
@@ -0,0 +1,185 @@
+order: 1
+id: powerpoint-text-get-set-textrange
+name: Work with text range selections
+description: 'Get, set, load, and save text range selections.'
+host: POWERPOINT
+api_set:
+    PowerPointApi: '1.5'
+script:
+    content: |
+        $("#getSelectedTextRange").click(() => tryCatch(getSelectedTextRange));
```

Comment:
The first button throws an error if no text is selected. Could we add a method + a button that creates and selects text, as the first step?

Reply:
I added a warning that you need to select some text.